pkg/services/librarypanels/librarypanels_test.go

```diff
@@ -1431,14 +1431,11 @@ func createDashboard(t *testing.T, sqlStore *sqlstore.SQLStore, user *models.Sig
 func createFolderWithACL(t *testing.T, sqlStore *sqlstore.SQLStore, title string, user *models.SignedInUser,
 	items []folderACLItem) *models.Folder {
 	t.Helper()
-
```

Comment:
could you add here the blank spaces, that it won't show in the PR

Reply:
added!